src/giftCards/GiftCardUpdate/messages.ts

```diff
@@ -12,6 +12,21 @@ export const giftCardUpdateDetailsCardMessages = defineMessages({
   }
 });
 
+export const giftCardUpdateActionBarMessages = defineMessages({
+  delete: {
+    defaultMessage: "Delete",
+    description: "delete button"
+  },
+  back: {
+    defaultMessage: "Back",
+    description: "back button"
+  },
+  save: {
+    defaultMessage: "Save",
+    description: "save button"
+  }
+});
+
```

Comment:
Let's use `import { commonMessages } from "@saleor/intl"` instead of defining new messages

Reply:
I didn't know such thing existed, thanks 😛